### PR TITLE
discovery: Rename Systemd to Cgroup

### PR DESCRIPTION
**Pull request by javierhonduco**

### CODE REVIEW

**Comment by kakkoyun** [comment]
Just a reminder in parallel to this, we need to update the parca.dev documentation.

File: pkg/discovery/cgroup.go [hunk 1]
```diff
@@ -123,7 +123,7 @@ func (c *SystemdDiscoverer) ReconcileUnit(ctx context.Context, unit string) (mod
 		return model.LabelSet{agent.CgroupPathLabelName: model.LabelValue(p)}, nil
 	}
 
-	f, err := os.Open(fmt.Sprintf("/sys/fs/cgroup/systemd/system.slice/%s/cgroup.procs", unit))
+	f, err := os.Open(fmt.Sprintf("/sys/fs/cgroup/cgroup/system.slice/%s/cgroup.procs", unit))
```
Comment (javierhonduco) [inline pkg/discovery/cgroup.go:126]: Note to self, fix this

**Comment by javierhonduco** [comment]
Closing for now, will re-open it after landing system-wide

**Comment by javierhonduco** [comment]
Plot twist: this all got removed in https://github.com/parca-dev/parca-agent/pull/627 as it's no longer necessary

**Pull request closed without merging**